src/server/worldserver/worldserver.conf.dist

```diff
@@ -796,7 +797,7 @@ CharactersPerRealm = 12
 #
 #    HeroicCharactersPerRealm
 #        Description: Limit number of heroic class characters per account on this realm.
-#        Range:       1-10
+#        Range:       1-12
 #        Default:     1
```

Comment:
I **could** but, if I am not mistaken, you would need to separate these checks...
Just changing `Default: 1` to `Default: 2` would enable people to create 2 DKs or 2 DHs. This config (`HeroicCharactersPerRealm`) needs to be redesigned and split into two separate ones. Example:

```
DeathKnightCharactersPerRealm
Description: Limit number of Death Knight characters per account on this realm.
Range:       1-12
Default:     1
```

```
DemonHunterCharactersPerRealm
Description: Limit number of Demon Hunter characters per account on this realm.
Range:       1-12
Default:     1
```